app/models.py

```diff
@@ -1986,16 +1985,16 @@ class DateTimeDimension(db.Model):
     bst_date = db.Column(db.Date, nullable=False, primary_key=True, index=True)
     year = db.Column(db.Integer(), nullable=False)
     month = db.Column(db.Integer(), nullable=False)
-    month_name = db.Column(db.Text(), nullable=False)
+    month_name = db.Column(db.String(), nullable=False)
```

Comment:
Why are we going back and forth between string and text? Also, "month name" should absolutely have a limited amount of characters.